frontend/components/notifications/management/modal/NotificationsManagementModalWebhook.vue

```diff
@@ -0,0 +1,168 @@
+<script lang="ts" setup>
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import {
+  faPaperPlane
+} from '@fortawesome/pro-solid-svg-icons'
+import { useForm } from 'vee-validate';
+import { object, string, boolean } from "yup";
+import { API_PATH } from '~/types/customFetch';
+
+const { props, close } = useBcDialog<{ webhookUrl: string, hasWebhookDiscord: boolean }>()
+
+const { t: $t } = useI18n()
+
+const { defineField, errors, handleSubmit, setFieldError, meta, values} = useForm({
+  validationSchema: object({
+    webhookUrl: string().url($t('validation.url.invalid')),
+    hasWebhookDiscord: boolean(),
+  }),
+})
+
+const [webhookUrl, webhookUrlAttrs] = defineField('webhookUrl', {
+  validateOnModelUpdate: false,
+})
+
+const [hasWebhookDiscord, hasWebhookDiscordAttrs] = defineField('hasWebhookDiscord', {
+  validateOnModelUpdate: false,
+})
+
+watch(props, () => {
+  webhookUrl.value = props.value?.webhookUrl || ''
+  hasWebhookDiscord.value = props.value?.hasWebhookDiscord || false
+})
+
+const isFormDirty = computed(() => meta.value.dirty)
+const isFormValid = computed(() => meta.value.valid)
+// const isFormEmpty = computed(() => !webhookUrl.value && !hasWebhookDiscord.value)
+
+const toast = useBcToast()
+const { fetch } = useCustomFetch()
+const handleTestNotification = async () => {
+  // 1. could not be implemented as a custom validation rule, 
+  // as they are always triggerd onMounted (at cast time)
+  if(!webhookUrl.value && hasWebhookDiscord.value) { //1. 
+    setFieldError('hasWebhookDiscord', $t('validation.webhook.discord_empty'))
+    return
+  }
+  if(!webhookUrl.value && !hasWebhookDiscord.value) {
+    // could not be implemented as a custom validation rule, 
+    // as they are always triggerd onMounted (at cast time)
+    setFieldError('webhookUrl', $t('validation.webhook.empty'))
+    return
+  }
+  if(!isFormValid.value) {
+    return
+  }
+  try {
+    if(hasWebhookDiscord.value) {
+      console.log(`hello`)
```

Comment:
please don't leave any console.log's in the code when you push it - only use it for short term local debugging ;-)